uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-api/src/main/java/org/uberfire/ext/layout/editor/api/PerspectiveServices.java

```diff
@@ -20,17 +20,19 @@
 
 import org.jboss.errai.bus.server.annotations.Remote;
 import org.uberfire.backend.vfs.Path;
+import org.uberfire.ext.editor.commons.file.DefaultMetadata;
 import org.uberfire.ext.editor.commons.service.support.SupportsCopy;
 import org.uberfire.ext.editor.commons.service.support.SupportsDelete;
 import org.uberfire.ext.editor.commons.service.support.SupportsRename;
+import org.uberfire.ext.editor.commons.service.support.SupportsSaveAndRename;
 import org.uberfire.ext.layout.editor.api.editor.LayoutTemplate;
 import org.uberfire.ext.plugin.model.Plugin;
 
 @Remote
-public interface PerspectiveServices
-        extends SupportsCopy,
-                SupportsRename,
-                SupportsDelete {
+public interface PerspectiveServices extends SupportsCopy,
+                                             SupportsDelete,
+                                             SupportsRename,
```

Comment:
I don't think you need to extend ```SupportsRename``` as you're extending ```SupportsSaveAndRename```.

Reply:
Done.